src/app/components/Curation/index.test.tsx

```diff
@@ -134,4 +134,26 @@ describe('Curation', () => {
       ).not.toBeInTheDocument();
     });
   });
+
+  describe('Headings', () => {
+    it('should render correctly when there are multiple curations and the curation only has 1 summary', () => {
```

Comment:
I spotted an issue whereby if there were multiple curations on the page, but one of the curations only had 1 summary, then the curation title was being displayed as a h2 as well as the first summary title (which is incorrect). 

To see this broken, use the View Document Outline feature on Web Developer tools on https://5d28eb3fe163f6002046d6fa-emazeoibkn.chromatic.com/iframe.html?args=&id=pages-home-page--kyrgyz&viewMode=story

This is incorrect - Pidgin test 2 - Title should be nested (as a h3) within Curation with Live Promo
![image](https://github.com/bbc/simorgh/assets/58214768/f2b8a7a6-a232-486f-aca9-7ee04692149b)
:

Reply:
This fix is visible here: https://wsteama-839-live-promo-fixture-data--5d28eb3fe163f6002046d6fa.chromatic.com/iframe.html?args=&id=pages-home-page--kyrgyz&viewMode=story